packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -281,3 +285,23 @@ export default async function processDeployment({
     throw err;
   }
 }
+
+export const archiveSuggestionText =
+  'Try using `--archive=tgz` to limit the amount of files you upload.\nLearn more: https://vercel.com/docs/cli/deploy#archive';
```

Comment:
Rather than hard-coding the "Learn more:" part, let's set the `err.link` property and ensure we're rendering the error through [`output.prettyError()`](https://github.com/vercel/vercel/blob/main/packages/cli/src/util/output/create-output.ts#L126-L146).

Reply:
Looks much nicer now!
![image](https://github.com/user-attachments/assets/95f2c647-a0bd-4b03-94b8-9d210a30694e)